plugins/ComposeAuthUI/src/commonMain/kotlin/io/github/jan/supabase/compose/auth/ui/annotations/AuthUiExperimental.kt

```diff
@@ -0,0 +1,7 @@
+package io.github.jan.supabase.compose.auth.ui.annotations
+
+/**
+ * Used to mark experimental APIs
+ */
+@RequiresOptIn(level = RequiresOptIn.Level.ERROR, message = "This API is experimental and may not be stable yet")
+annotation class AuthUiExperimental
```

Comment:
The documentation could be more specific, something like this:
```suggestion

/**
 * Used to mark experimental Compose Auth Ui APIs
 */
@RequiresOptIn(level = RequiresOptIn.Level.ERROR, message = "This API is experimental and may not be stable yet")
annotation class AuthUiExperimental
```

Reply:
Thanks!